plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/S3BlobStoreContainerTests.java

```diff
@@ -33,6 +33,9 @@
 package org.opensearch.repositories.s3;
 
 import org.mockito.ArgumentCaptor;
+import org.opensearch.action.ActionListener;
```

Comment:
Changes in this file will be added as part of https://github.com/opensearch-project/OpenSearch/pull/8314. Reviewers can ignore these changes in this PR.